integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -589,24 +413,15 @@ public int flush(String path, FuseFileInfo fi) {
   }
 
   private int flushInternal(String path, long fd) {
```

Comment:
many of these internal functions can be just inlined as lambda. It's up to you. Generally speaking if it's only a few lines it's easier to read if they are inlined as lambda. But here for consistency keeping them all as separate function is fine too. Just be aware of the options.

Reply:
Thanks, nice suggestion! this can be done in separate PR to avoid this PR being too long.